core/templates/dev/head/css/oppia.css

```diff
@@ -112,127 +112,190 @@ body {
   margin-bottom: 30px;
 }
 
-.oppia-profile-top-card {
-  background-color: #fff;
-  margin: 0 0 24px 0;
-  padding: 24px;
-}
-
-.oppia-profile-subject-interest-container {
-  font-weight: bold;
-  margin-top: 16px;
-}
-
-.oppia-profile-subject-interest {
-  background-color: rgba(0, 150, 136, 0.85);
-  border-radius: .25em;
-  color: #fff;
-  display: inline;
-  font-size: 0.75em;
-  line-height: 1;
-  margin: 0 2px;
-  padding: .3em .35em .3em .5em;
-  text-align: center;
-  vertical-align: baseline;
-  white-space: nowrap;
+.oppia-profile-container {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 60px;
+  margin-bottom: 30px;
+  max-width: 980px;
+  width: 90%;
 }
 
-.oppia-profile-no-interests-text {
-  color: #888;
-  font-weight: normal;
+.oppia-profile-user-card {
+  background-color: #fff;
+/*  border: 2px solid #a9a9a9;*/
+  float: left;
+  margin-right: 28px;
+  padding: 24px;
+  width: 220px;
 }
 
 .oppia-profile-picture-fullsize {
   /* This maintains the aspect ratio of the picture. */
```

Comment:
Not sure which line in here is being referred to by this comment. If you know, can you put the comment just above that line?

If you don't know, I think it's fine to just drop the comment entirely.